assets/css/style.css

```diff
@@ -1,70 +1,59 @@
-
-/** My custom CSS classes **/
-/** Add your custom classes here to make it easier to find **/
-/**@import "main.css"**/
-.hr-style{
+.hr-style {
     margin-left:15%;
     margin-right:15%;
     margin-top: 1%;
-  }
-  
-.visited {
+}
+.a {
```

Comment:
This class isn't really representative. I'm not 100% sure the role of it, but it could be called something more along the lines of `header-title`.